openBEM.js

```diff
@@ -865,21 +850,18 @@ calc.temperature = function (data) {
         }
     }
 
-    var Te = [];
-    for (var m = 0; m < 12; m++) {
-        //Te[m] = datasets.table_u1[data.region][m] - (0.3 * data.altitude / 50);
-        Te[m] = datasets.table_u1[data.region][m];
-    }
-
     //----------------------------------------------------------------------------------------------------------------
     // 7. Mean internal temperature (heating season)
     //----------------------------------------------------------------------------------------------------------------
 
     // Bring calculation of (96)m forward as its used in section 7.
     // Monthly average external temperature from Table U1
-    // for (var i=1; i<13; i++) data['96-'+i] = table_u1[i.region][i-1]-(0.3 * i.altitude / 50);
 
-    // See utilisationfactor.js for calculation
+    var Te = [];
```

Comment:
Moved this bit of code down below the "7. Mean internal temperature (heating season)" comment, as this seems to be the bit of code that is actually relevant to the comment.